ecms-social-integration/src/main/java/org/exoplatform/wcm/notification/plugin/FileActivityChildPlugin.java

```diff
@@ -52,13 +52,14 @@
 public class FileActivityChildPlugin extends AbstractNotificationChildPlugin {
   private static final Log   LOG                          = ExoLogger.getLogger(FileActivityChildPlugin.class);
   public final static ArgumentLiteral<String> ACTIVITY_ID = new ArgumentLiteral<String>(String.class, "activityId");
-  public final static String PRIVATE_FOLDER_PATH          = "/Private/";
+  public final static String PRIVATE_FOLDER_PATH          = "/private/rest";
```

Comment:
I don't think this is correct, this variable is for the name of Private folder and not rest URL